README.md

```diff
@@ -1,8 +1,6 @@
-# SPORTTIA
-
+ 
 ##  **Idea  Principal**
 
-Hacer la reserva de las pistas es mucho más fácil hacerlo online tanto para el cliente como para el trabajador ya que ahorra mucho tiempo y no necesitas trasladarte, sin embargo hay muchos sitios que no cuentan con este sistema, por eso me he propuesto hacer esta aplicación.
+Cuando eres seguidor de varios videojuegos, a veces cuesta estar informado de cuándo lanzan un nuevo título de una saga, descuentos, y la posibilidad de reservar tus juegos favoritos, etc. Con este proyecto podremos estar informados sin la necesidad de recorrer Google.
```

Comment:
Por favor, mira la lista de comprobación en el guión del hito. ¿Qué lógica de negocio tiene? ¿De qué forma esto hace algo más que almacenar y buscar? No os quedéis con el simple hecho binario de está bien o está mal. Debéis intentar entender qué está bien o mal y tratar de corregirlo (en uno nuevo o en el que tuviérais antes).